src/domain/entities/file.ts

```diff
@@ -0,0 +1,66 @@
+import type { Note } from './note.js';
+import type User from './user.js';
+
+/**
+ * File types for storing in object storage
+ */
+export enum FileTypes {
+  /**
+   * @todo define types
+   */
+}
+
+/**
+ * Interface representing a file entity
+ */
+export default interface UploadedFile {
+  /**
+   * File identifier
+   */
+  id: number;
+
+  /**
+   * File unique hash which stores in some object storage
+   */
+  key: string;
+
+  /**
+   * User who uploaded the file
+   */
+  userId: User['id'];
```

Comment:
do we really need this? i think that cover would be generated and uploaded without user

Reply:
made optional